data/hai/hai reveal 4 middle.txt

```diff
@@ -1141,28 +1141,54 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			choice
+				`	"You can trust the Unfettered to help you, but you must be careful to avoid a war with the humans. That would be catastrophic for everyone."`
+				`	"The Unfettered have offered their help because you would not help your people. If you change your decision, you will not need their involvement."`
+					goto "turn back unfettered"
+			action
+				set "told hai to accept unfettered help"
+			`	One Elder rises and says, "The Unfettered only know how to make war. Once they are in human space, there's no telling what they'll do." Many in the room nod in agreement.`
+			`	Teeneep responds, "They are clever strategists and dedicated in our protection. Never would they take such unnecessary risks. I am certain they would accept an overseer among us to advise them on limits to their actions."`
+				goto "the elders vote"
+			label "turn back unfettered"
+			action
+				set "told hai to reject unfettered help"
+			`	A disgruntled Elder responds, "Change our decision? Our decision not to violate human sovereignty? Even if I lose my Eldership, I will not vote for violence."`
+			`	Teeneep says, "If you take no action, and refuse help from our brethren, they'll pass through our territory anyway. We can hold back Unfettered raids, but their entire armada? That is the unnecessary violence you bring through inaction."`
+			label "the elders vote"
+			`	Another Elder interjects, "You're exaggerating the impact, Teeneep. Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
+			`	Many Elders seem appalled by that observation, but the moderator interrupts. "We will vote on two counts: Do we act? Do we accept Unfettered help?"`
+			choice
+				`	(Await the vote.)`
+			`	After a few more minutes of private discussions, the vote is tallied. Unfortunately, the majority vote "no" on both counts. Your words may not have swayed the vote, but perhaps they'll resonate in the Hai psyche long enough to have an effect.`
+			`	Teeneep is allowed to respond, and says ominously, "You know what will happen next."`
+			`	An angry Elder replies, "You wouldn't dare!" to which Teeneep responds, "It is already scheduled to happen unless I actively stop it!" She smiles, a little self indulgently, and withdraws from the room. Sayari follows, motioning for you to do so as well.`
+			`	Sayari leads you to a private meeting room.`
+			`	Teeneep explains her ominous words. "Our democracy allows a Petition of No Confidence against the Elders. If there are enough petitioners, it forces an election. Activists in my organization have been pushing a petition for weeks.`
+			`	"It was obvious what the Elder Council's response would be," and with a sarcastic smile, she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, I arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
```

Comment:
```suggestion
			`	"It was obvious what the Elder Council's response would be," and with a sarcastic smile she adds, "and I knew I would only come back alive if the Unfettered agreed to help. So, I arranged in advance for the petition to be promoted along with a widespread broadcast of these proceedings."`
```
I'd like to try get rid of this repetition with the smiling.

Reply:
I changed it to "with a bit of sarcasm"